content/network/drivers/overlay.md

```diff
@@ -74,215 +49,55 @@ $ docker network create -d overlay --attachable my-attachable-overlay
 You can specify the IP address range, subnet, gateway, and other options. See
 `docker network create --help` for details.
 
-### Encrypt traffic on an overlay network
-
-All swarm service management traffic is encrypted by default, using the
-[AES algorithm](https://en.wikipedia.org/wiki/Galois/Counter_Mode) in
-GCM mode. Manager nodes in the swarm rotate the key used to encrypt gossip data
-every 12 hours.
+## Encrypt traffic on an overlay network
 
-To encrypt application data as well, add `--opt encrypted` when creating the
-overlay network. This enables IPSEC encryption at the level of the vxlan. This
-encryption imposes a non-negligible performance penalty, so you should test this
-option before using it in production.
```

Comment:
The specs of the encryption used (IPSec with AES-GCM cipher) is important information for users to decide whether it is suitable for their use case. And there is still a performance penalty that users should be aware of.

Reply:
Just to check - "AES-GCM" doesn't appear in the new text here... was this duplicated info, so still mentioned somewhere else in the docs?